Logging/src/PsrLogger.php

```diff
@@ -430,6 +430,22 @@ public function getMetadataProvider()
      * @access private
      */
     public function serialize()
+    {
+        return serialize($this->__serialize());
+    }
+
+    /**
+     * Unserializes data.
+     *
+     * @param string
+     * @access private
+     */
+    public function unserialize($data)
+    {
+        $this->__unserialize(unserialize($data));
+    }
+
+    public function __serialize(): array
```

Comment:
We cannot have return types because this library is still compatible with PHP 5.5
```suggestion
    public function __serialize()
```